src/core_plugins/kibana/public/dashboard/panel/_dashboard_panel.scss

```diff
@@ -24,64 +24,47 @@
   /**
    * 1. We want the doc-table-container to scroll only when embedded in a dashboard panel
    * 2. Fix overflow of vis's specifically for inside dashboard panels, lets the panel decide the overflow
+   * 3. Force a better looking scrollbar
    */
   .doc-table-container {
     flex: 1 1 0; /* 1 */
     overflow: auto; /* 1 */
   }
 
+  .visualization {
+    @include euiScrollBar; /* 3 */
+  }
+
   .visualization .vis-container {
     overflow: visible; /* 2 */
   }
 }
 
-// CONTENT
+.dshPanel--editing {
+  transition: all $euiAnimSpeedFast $euiAnimSlightResistance;
 
-.dshPanel__panel {
-  background-color: $euiColorEmptyShade;
-  border: 1px solid $euiColorEmptyShade;
-  overflow: hidden;
-
-  // maintain the 100% height of the panel
-  height: 100%;
-  flex: 1;
-
-  // flex layout allows us to define the visualize element as "fill available space"
-  display: flex;
-  flex-direction: column;
-  justify-content: flex-start;
-}
-
-.dshPanel__panel--editing {
-  // SASSTOD: Change this back $euiBorderColor, when EUI adds !default to borders
-  border-color: $euiColorLightShade;
-
-  &:hover {
+  &:hover,
+  &:focus {
     border-color: $euiColorPrimary;
+    @include euiSlightShadowHover;
   }
 }
 
-// Adjust borders/margin/etc... for spaced out panels
-.dshLayout-withMargins {
-  .dshPanel__panel {
-    @include euiBottomShadowSmall;
-    // SASSTOD: Change this back $euiBorderColor, when EUI adds !default to borders
-    border-color: $euiColorLightShade;
-    border-radius: $euiBorderRadius;
-  }
+// LAYOUT MODES
 
-  .dshPanel__panel--editing {
-    transition: all $euiAnimSpeedFast $euiAnimSlightResistance;
+// Adjust borders/etc... for non-spaced out panels
+.dshLayout-withoutMargins {
```

Comment:
I saw one more thing, the panel styles should be removed when in expanded mode as well. I'd change lines 55-56 to be:
```
// Adjust borders/etc... for non-spaced out and expanded panels
.dshLayout-withoutMargins,
.dshDashboardGrid__item--expanded {
```